src/styles/ftva/_section-wrapper.scss

```diff
@@ -47,19 +50,64 @@
 
             background-color: var(--color-theme);
 
-
             :deep(*) {
                 .block-highlight.card {
                     background-color: var(--color-children-theme);
                 }
             }
-
-
         }
 
         // FTVA uses different page max-width
         >:deep(*) {
             max-width: var(--ftva-container-max-width);
         }
     }
-}
\ No newline at end of file
+
+    // Breakpoints
+    @media #{$medium} {
+        .section-header {
+            margin-bottom: 10px;
+        }
+
+        .section-link {
+            margin-bottom: 20px;
+        }
+
+        &.theme-paleblue {
+            --color-theme: #{$page-blue};
+
+            --color-children-theme: &.theme-paleblue {
```

Comment:
this is wrong syntax
Update the whole scss file with following code
`
.ftva.section-wrapper {
    .section-header {
        // Set to zero to remove margin under Section Title
        // Explore Other Series
        margin-bottom: 0px;
    }

    // default link styles for FTVA
    .section-link {
        top: 11px;
        @include ftva-button-link;
        color: #{$medium-grey};

        text-decoration: underline;
        text-decoration-thickness: 3px;
        -webkit-text-decoration-color: #{$grey-blue};
        text-decoration-color: #{$grey-blue};
        text-underline-offset: 4px;

        &:hover {
            -webkit-text-decoration-color: #{$bright-blue};
            text-decoration-color: #{$bright-blue};
        }
    }

    &.top-level {
        &.theme-white {
            &.section-banner {
                >:deep(*) {
                    max-width: var(--ftva-container-max-width);
                }
            }

            .section-featured-banner {
                max-width: var(--ftva-container-max-width);
            }

            :deep(.section-teaser-card) {
                --color-children-theme: #{$page-blue};

                .block-highlight.card {
                    background-color: var(--color-children-theme);
                }
            }
        }

        &.theme-paleblue {
            --color-theme: #{$page-blue};
            --color-children-theme: var(--color-white);

            background-color: var(--color-theme);

            :deep(*) {
                .block-highlight.card {
                    background-color: var(--color-children-theme);
                }
            }
        }

        // FTVA uses different page max-width
        >:deep(*) {
            max-width: var(--ftva-container-max-width);
        }
    }

    // Breakpoints
    @media #{$medium} {
        .section-header {
            margin-bottom: 10px;
        }

        .section-link {
            margin-bottom: 20px;
        }

        &.theme-paleblue {
            --color-theme: #{$page-blue};
            --color-children-theme: var(--color-white);

            :deep(*) {
                .section-teaser-list {
                    .list {
                        background-color: var(--color-theme);
                    }

                    .block-card-three-column {
                        .meta {
                            background-color: var(--color-children-theme);
                        }
                    }
                }
            }
        }
    }
}
`